charts/opentelemetry-demo-webstore/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+type: application
+name: opentelemetry-demo-webstore
+version: 0.1.0
+description: opentelemetry demo webstore helm chart
+home: https://opentelemetry.io/
+sources:
+  - https://github.com/open-telemetry/opentelemetry-demo-webstore
+maintainers:
+  - name: wph95
```

Comment:
I took a look at the other charts' maintainers list and realized they are pretty out of date, assuming this list is supposed to be the maintainer of the chart and not the thing being installed.

Can you add myself and @dmitryax to this list?

Reply:
cool! added